File: Source/UserManagement/Domain/DataCollector/PhoneNumber/AddPhoneNumberToDataCollector.cs
using System;
using System.Collections.Generic;
using Concepts;
using Dolittle.Commands;

namespace Domain.DataCollector.PhoneNumber
{
    public class AddPhoneNumberToDataCollector : ICommand
    {
        public DataCollectorId DataCollectorId { get; set; }
        public string PhoneNumber { get; set; }
    }
}

File: Source/UserManagement/Web.Angular/src/app/domain/data-collector/AddPhoneNumberToDataCollector.ts
import { Command } from '../../services/Command';

export class AddPhoneNumberToDataCollector extends Command {
    dataCollectorId: string;
    phoneNumber: string;

    constructor() {
        super();
        this.type = 'CBS#UserManagement.DataCollector.PhoneNumber-AddPhoneNumberToDataCollector+Command@1';
    }
}
